putio.py

```diff
@@ -9,6 +9,15 @@
 import requests
 import iso8601
 
+KB = 1024
+MB = 1024 * KB
+
+# Downloads are broken up into ranges of this size
+RANGE_SIZE = 10 * MB
```

Comment:
Why do we need to make range requests of fixed 10M size? Can't we just continue from where we left of?
```
Range: bytes=12345-
```